Marks activateService and activateServicePlan public APIs as deprecated. (#15534)
microsoftgraph/microsoft-graph-docs: api-reference/beta/api/user-activateserviceplan.md

doc_type: apiPageType
---

# user: activateServicePlan (deprecated)

Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

> [!CAUTION]

> > The user `activateServicePlan` API will be deprecated and will stop return data on March 15, 2022.

Reviewer: ```suggestion
> The **activateServicePlan** API is deprecated and will stop returning data on March 15, 2022.
```
